wire/internal/wire/wire_test.go

```diff
@@ -70,7 +70,8 @@ func TestWire(t *testing.T) {
 	for _, test := range tests {
 		test := test
 		t.Run(test.name, func(t *testing.T) {
-			t.Parallel()
+			// TODO(light): These tests should be parallelizable, but seems to flake.
```

Comment:
s/but seems to flake/but they have tended to flake when run that way/.

Reply:
Done.